collect_app/src/main/java/org/odk/collect/utilities/Clock.java

```diff
@@ -0,0 +1,10 @@
+package org.odk.collect.utilities;
```

Comment:
This is potentially controversial. It feels like it might be nice to start pulling objects like this out of the `android` package so we can have a distinction in our code what things actually relate to the Android world. This is useful for drawing a line around components and trying not to muddle them with Android stuff. The main advantage other than seperation of concerns and simiplifcation is to let us write tests for these kinds of things without using Robolectric.